src/webgl/3d_primitives.js

```diff
@@ -468,6 +485,11 @@ const _truncatedCone = function(
  *   background(205, 105, 94);
  *   rotateY(millis() / 1000);
  *   cylinder(20, 75, detailX.value(), 1);
+ *   describe(`A spinning view of what appears to be a white plane.
+ *     Black lines on its surface show how the shape is built out of
+ *     many small triangles.
+ *     As you increase the slider below the canvas, the shape gradually
+ *     becomes smoother and closer to a cylinder.`);
```

Comment:
Can we make description more succinct?

Reply:
Same response as my [first comment](https://github.com/processing/p5.js/pull/5599#discussion_r815326608).